src/algebra/ring/basic.lean

```diff
@@ -765,7 +765,6 @@ assume h,
 calc
   (a - b) * e + c = (a * e + c) - b * e : begin simp [sub_mul, sub_add_eq_add_sub] end
               ... = d                   : begin rw h, simp [@add_sub_cancel α] end
-
 end non_unital_non_assoc_ring
```

Comment:
Adding back the blank line might speed up the build, and there's no reason to remove it anyway
```suggestion

end non_unital_non_assoc_ring
```